package.json

```diff
@@ -121,7 +125,8 @@
       "/node_modules/(?!(@yext/search-headless-react)/)"
     ],
     "moduleNameMapper": {
-      "./SearchCore": "<rootDir>/tests/__fixtures__/core/SearchCore.ts"
+      "./SearchCore": "<rootDir>/tests/__fixtures__/core/SearchCore.ts",
+      "\\.(css|less|scss|sass)$": "identity-obj-proxy"
```

Comment:
for my own understanding, why is this needed?

Reply:
This is added because storyshot needs to read .storybook folder for configuration and could not read the index.css file

Reply:
I ran into this issue on my own PR-- but I had to come to a similar solution.  When I imported a CSS file into one of my components, Jest tests started to fail because it seemed like Jest did not know what to make out of the CSS class imports, so you have to configure Jest to know what to do with them.

Some more information!
https://jestjs.io/docs/webpack 
https://github.com/facebook/jest/issues/3094

Reply:
But, I'm not clear on why it's needed here in this situation, so Juliann could probably answer more on that